doc/contributors/releasegovernance.md

```diff
@@ -0,0 +1,46 @@
+# Release Governance
+
+The goal of this document is to define the method and schedule of Operator Lifecycle Manager releases.
+
+## Scope
+
+- Define a consistent release cadence for the project
+- Provide a framework to ensure consistent high quality releases
+- Define a consistent understanding of what is in scope for a release
+- Ensure community members understand the release cycle
+
+## Roles and Organization
+
+The Operator Lifecycle Manager project is part of the Operator Framework. As a result, it's community membership guidelines adhere to the definitions defined in the Operator Framework's [community membership definition](https://github.com/operator-framework/community/blob/master/community-membership.md). For the purpose of this document, there are two roles that are relevant for the release schedule.
+
+- Project reviewers provide guidance on whether or not a release is ready. Before a release can be cut, we will require that two reviewers provide the `/lgtm` label to the release.
+- Project approvers provide the final approval on whether or not a release is ready to be cut. A quorum (greater than 50%) of the project's approvers need to agree that a release is ready by writing the `/approve` label to the release.
```

Comment:
This seems like a lot of people to simply cut a minor release version, no? We're talking about 5 people at a minimum. If the number of approvers grows by 1, then a quorum would be 4 which means 6 people total. Seems like a lot of coordination for a go/no-go on a release. 

Do we really need so much review for the upstream community to cut a release? We want more frequent releases and for upstream members to be able to release on an accelerated schedule. I would think 2 `lgtms` and 1 `approve` would suffice? With the goal of the upstream community members being able to cut their own releases eventually with a different set of owners